sklearn/preprocessing/tests/test_callable_transformer.py

```diff
@@ -0,0 +1,79 @@
+from nose.tools import assert_equal
+import numpy as np
+
+from ..callable_transformer import CallableTransformer
+
+
+def _make_func(args_store, kwargs_store, func=lambda X, *a, **k: X):
+    def _func(X, *args, **kwargs):
+        args_store.append(X)
+        args_store.extend(args)
+        kwargs_store.update(kwargs)
+        return func(X)
+
+    return _func
+
+
+def test_delegate_to_func():
```

Comment:
Would these tests simplify if dropping support for `args` and `kwargs` in favor of `partial`?